src/commands/codepush/lib/react-native-utils.ts

```diff
@@ -468,11 +468,51 @@ function parseBuildGradleFile(gradleFile: string) {
     throw new Error(`Unable to find gradle file "${buildGradlePath}".`);
   }
 
+  return buildGradlePath;
+}
+
+function parseBuildGradleFile(gradleFile: string) {
+  const buildGradlePath: string = getBuildGradleFilePath(gradleFile);
   return g2js.parseFile(buildGradlePath).catch(() => {
     throw new Error(`Unable to parse the "${buildGradlePath}" file. Please ensure it is a well-formed Gradle file.`);
   });
 }
 
+// parse gradle.properties file
+async function parseGradlePropertiesFile(gradleFile: string) {
+  // find gradle.properties file based on gradleFile
+  const buildGradlePath: string = getBuildGradleFilePath(gradleFile);
+  const gradlePropertiesPath: string = path.join(path.dirname(buildGradlePath), "../", "gradle.properties");
+  // check if gradle.properties file exists
+  if (fileDoesNotExistOrIsDirectory(gradlePropertiesPath)) {
+    console.error(`Unable to find gradle.properties file at "${gradlePropertiesPath}".`);
+    return;
```

Comment:
In this case, it is better to write that we return null

Reply:
Okey. ` return undefined ` now.